backend/src/app/rpc/queries/teams.clj

```diff
@@ -229,3 +230,20 @@
 (defn retrieve-team-stats
   [conn team-id]
   (db/exec-one! conn [sql:team-stats team-id team-id]))
+
+
+;; --- Query: Team invitations
+
+(s/def ::team-id ::us/uuid)
+(s/def ::team-invitations
+  (s/keys :req-un [::profile-id ::team-id]))
+
+(def sql:team-invitations
+  "select email_to as email, role, (valid_until < ?) as expired from team_invitation where team_id = ?")
+
+
+(sv/defmethod ::team-invitations
+  [{:keys [pool] :as cfg} {:keys [profile-id team-id]}]
+  (with-open [conn (db/open pool)]
+    (check-read-permissions! conn profile-id team-id)
+    (db/exec! conn [sql:team-invitations (dt/now) team-id])))
```

Comment:
I think is better use the `now()` directly in SQL, so you don't need to pass the `(dt/now)` as parameter. But maybe I'm wrong.

Reply:
I think it is more consistent to always calculate the dates on the same place (database or back code). The token expiration date is set on back as (dt/in-future "48h"). I think calculating the date on the same place can avoid time discrepanciesand/or timezones issues...

Reply:
Finally changed to now()